shared/middleware/helpers/auth_helpers.rb

```diff
@@ -23,6 +23,17 @@ def admin?
   current_user && !!current_user[:admin]
 end
 
+# @param [String] permission - Name of the permission we're interested in
```

Comment:
I can try. Not sure offhand if that's trivial to do (going across the boundary between shared/dashboard)